src/webview/js/rewriteHtml.js

```diff
@@ -58,11 +58,37 @@ export const rewriteImageUrls = (auth: Auth, element: Element | Document = docum
   });
 };
 
+/* Rewrite the 'time' elements to make them more readable.
+ *
+ * 1. Changes text to localized human readable date-time.
+ * 2. Adds 'original-text' attribrute, to show original text alert dialog on press.
+ */
+const rewriteTime = () => {
+  const timeElements: NodeList<HTMLTimeElement> = document.querySelectorAll('time');
+  timeElements.forEach(elem => {
+    const timeStamp = elem.getAttribute('datetime');
+    const text = elem.innerText;
+
+    // 'text' and 'timeStamp' are guaranteed to be present.
+    if (text == null || timeStamp == null) {
+      return;
+    }
+
+    const d = new Date(timeStamp);
+    elem.setAttribute('original-text', text);
+    elem.innerText = `🕒  ${d.toLocaleString(undefined, {
```

Comment:
If you pass `undefined`, I wonder if it's smart enough to detect the correct locale, e.g., for putting the month before the day or vice versa: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#Parameters

I think maybe it is; this is running in a WebView, which the device has some control over—hopefully enough to tell it the appropriate device-level setting.